backend/schema/metadataingress.go

```diff
@@ -79,7 +79,7 @@ type IngressPathComponent interface {
 type IngressPathLiteral struct {
 	Pos Position `parser:"" protobuf:"1,optional"`
 
-	Text string `parser:"@Ident" protobuf:"2"`
+	Text string `parser:"@~Whitespace+" protobuf:"2"`
```

Comment:
I think this will need to be something like `@~(Whitespace | '/' | '{' | '}')+`, otherwise it will just keep consuming, including path parameters?

Reply:
ah mixed up literal and the path parameter matchers. updated to `@~(Whitespace | '/' | '{' | '}')+` for literal and leaving as `'{' @Ident '}'` for the path param - does that sound right?